lib/stm32wba/hci/README

```diff
@@ -117,6 +117,9 @@ Patch List:
 			  app_conf.h
 			  scm.c
 
+	* Including common interfaces of MAC only and BLE/COEXISTENCE APIs
+	  Added file: lib/stm32wba/hci/ll_intf_cmn.h
+
```

Comment:
Instead of adding this, please update the file list above.

Reply:
done